reana_job_controller/htcondorcern_job_manager.py

```diff
@@ -93,8 +93,6 @@ def execute(self):
         job_ad['TransferOutput'] = '.'
         job_ad['PeriodicRelease'] = classad.ExprTree('(HoldReasonCode == 35)')
         job_ad['MaxRunTime'] = 3600
-        job_ad['Requirements'] = classad.ExprTree(
-            'TARGET.HasDocker && (OpSysAndVer =?= "CentOS7")')
```

Comment:
Can you add a small fix for a trouble I see with `master` branch?  

```
htcondorcern_job_manager.py:            sleep(10)
```

Needs `time.sleep()` to avoid import troubles...